pyomo/solvers/tests/solvers.py

```diff
@@ -274,13 +274,6 @@ def test_solver_cases(*args):
             io_options={"skip_objective_sense": True},
         )
 
-        _test_solver_cases['glpk', 'python'] = initialize(
-            name='glpk',
-            io='python',
-            capabilities=_glpk_capabilities,
-            import_suffixes=[],
-        )
-
```

Comment:
Why remove this?

Reply:
The GLPK direct interface (i.e., `solver_io=python`) was removed years ago.  This just generates errors when we check the SolverFactory for it.